CONTRIBUTING.md

```diff
@@ -0,0 +1,24 @@
+## Target branch
+
+Pull requests should target the `main` branch. If the change is relevant for a hotfix release, make a note in the PR description so it can be cherry picked after merge.
+
+## Adding to CHANGELOG
+
+All changes focused on users of the released builds should be documented by writing a new entry to [CHANGELOG.md](CHANGELOG.md), under the unreleased version at the top.
```

Comment:
Changelog entries are to be written within the scope of a specific upcoming release. In practice, changelog entries should always go under the `unreleased` section of `changelog.md`. As such, changelog entries and the specific language used in them must be with respect to the previous release. If a feature changes multiple times in development, the existing entry should be edited rather than appended if it is still unreleased at the time of writing.